qa/tasks/cephfs/test_fragment.py

```diff
@@ -314,3 +314,88 @@ def _count_fragmented():
             lambda: _count_fragmented() > 0,
             timeout=30
         )
+
+class TestDirfragKillPoints(TestFragmentation, CephFSTestCase):
+    # Two active MDS
+    MDSS_REQUIRED = 4
```

Comment:
Why is it necessary to export/import for these tests? You should be just splitting/merging directories by modifying `mds_bal_split_size`/`mds_bal_merge_size`, right?

Reply:
I think I got it wrong then. Since the split is between two MDS, the set of killpoints on the MDS doing the split was taken as the exporter set and the one where the frag goes is taken as the importer set. Should it just iterate over one consolidated list of killpoints with no notion of a source and dest?

Reply:
@batrick Thoughts on this?

Reply:
There is a notification mechanism for splitting/merging as other MDS may have the directory fragment replicated. Testing killpoints during that notification is worthwhile.

Splitting and merging fragments do not overlap so there is no point doing an `n*m` matrix of configurations. You just need to do `n+m`.

Reply:
Ack.